hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/publishing/HttpMetaHeaders.java

```diff
@@ -0,0 +1,6 @@
+package pl.allegro.tech.hermes.frontend.publishing;
+
+public interface HttpMetaHeaders {
```

Comment:
hmm.. so this is a source Header for trace? it's a bit confusing to see this and then `Hermes-Trace-Id` in MessageMetadataHeaders